libs/auth/src/lib/components/register/register.component.html

```diff
@@ -14,36 +14,34 @@ <h1 class="w-full my-4 text-lg font-bold text-center">Mebli</h1>
                  name="email"
                  type="email"
                  [placeholder]="'email_placeholder' | transloco"
+                 pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$" 
                  required
                  autocomplete="email"
                  class="w-full px-5 py-3 mt-1 text-base placeholder-gray-300 transition duration-500 ease-in-out transform border border-blue-200 rounded-sm focus:outline-none focus:ring-2 focus:ring-blue-200 focus:ring-offset-2 focus:ring-offset-blue-200"
                  [(ngModel)]="email">
           <div [hidden]="emailModel.valid || emailModel.pristine"
                class="w-full text-red-500">
-            {{'email' | transloco}} {{'is_required' | transloco}}
+            {{'email' | transloco}} {{'is_required' | transloco}} und gültige E-Mail Adresse eingeben
```

Comment:
Alle Übersetzungen / Texte sind in deutsch und englisch in den Datein de.json bzw. en.json zu pflegen und anschließend über das Pattern {{'translation_key' | transloco}} im HTML zu verwenden. Bitte keine fest verdrahteten Texte im Template

Reply:
Erledigt